l1-contracts/test/validator-selection/setupEpoch.tree

```diff
@@ -0,0 +1,19 @@
+SetupEpochTest
+├── when the rollup is in genesis state
```

Comment:
Think we could get rid of a whole bunch of these by using the constructor to set some of these values 👀